packages/common/src/polyfills/version/release-version.ts

```diff
@@ -0,0 +1,37 @@
+// default return values
+const OLD_DEV = [-1 ,0 ,0];
+const DEV = [0, 0, 0]
+
+/**
+ * get the console release version
+ * 
+ * @returns a tupple with the console release version, on none release it will return 0, 0, 0,
+ *          on old development server, return -1, 0, 0
+ */
+export const getReleaseVersion = () => {
+    const releaseVersion = window['SERVER_FLAGS']?.releaseVersion;
+    const consoleVersion = window['SERVER_FLAGS']?.consoleVersion;
+
+    if (!releaseVersion || typeof releaseVersion !== 'string' || releaseVersion === '') {
+        // if we don't have releaseVersion try consoleVersion
+        if (consoleVersion && typeof consoleVersion === 'string' && consoleVersion.startsWith('v6.0.6-20')) {
+            return OLD_DEV;
+        }
+
+        return DEV;
+    }
+    const [major, minor, bugfix] = releaseVersion.split('.').map((v) => parseInt(v, 10));
+
+    return [major, minor, bugfix]
+}
+
+/**
+ * Specific version check based on getReleaseVersion
+ * 
+ * @returns true if version is 4.11 or lower
+ */
+export const isConsoleVersion_4_11 = () => {
```

Comment:
Actually it would be even safer to evaluate this once i.e.
```
export const IS_CONSOLE_VERSION_4_11 = isConsoleVersion_4_11();
```
Then we could switch between lib version and polyfills version in a similar way as we do with mocks i.e.:
```
export const useModal = IS_CONSOLE_VERSION_4_11 ? useModalPolyfill : useModalSDK;
```